src/data.toml

```
@@ -1,18 +1,22 @@
-ios_file_access = "When sideloading with a paid certificate the application will be signed using a wildcard App ID.\nThis effectively disables the file access capability.\nYou need to sign the application with a local dev cert instead (which generally uses an explicit App ID by default) if you want this functionality.\n\nKeep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
+ios_app_icons = "When sideloading with a paid certificate the application will be signed using a wildcard App ID.\nThis effectively breaks being able to change app icons.\nYou need to sign the application with a local dev cert instead if you want this functionality.\n\nPlease keep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
 addon_404 = "You are not supposed to visit the url directly since it does not serve anything that can be rendered. Copy the link and use it to install the add-on."
+not_support_channel = "This is not the support channel, please move to https://discord.com/channels/1196075698301968455/1210652540757090315."
+cloudsync_not_working = "You might be using an outdated version of Cloud Sync, as older versions don't work anymore.\n1. Make sure you have the latest version of Cloud Sync (available in the pins) available\n2. If that doesn't work, and you have **antied** installed, uninstall and reinstall it (https://angelix1.github.io/MP/angel/antied/)\n### (It is advised to use local backup to avoid accidental erasure of cloud data as it can't be recovered once overridden with a empty sync.)"
+android15_error = "Refer to https://discord.com/channels/1196075698301968455/1205101050739167272/1264529990997905442"
+fakeprofilesthemeandeffects_guide = "## How to use Fakeprofilesthemeandeffects plugin-\n\n1 - Select colour\n2 - Select colour\n3 - Select effect\n4 - Copy FPTE\n5 - Paste what you just copied into about me, on a new line\n-# Note: you *might* experience using gboard clipboard when pasting the PTE. Instead use hold to paste.\n [󠀀](https://imgur.com/paPTG3J)"
+free_nitro = "## List of plugins that provides *nitro features for free*-\n1. [Freemoji plugin.](<https://rico040.github.io/bunny-plugins/freemoji/>)\n2. [Realmoji plugin.](<https://redstonekasi.github.io/vendetta-plugins/realmoji/>)\n3. [Fake Profile Themes And Effects plugin.](<https://ryan-0324.github.io/vendetta-plugins/FakeProfileThemesAndEffects/>)\n4. [UserBG plugin.](<https://rico040.github.io/bunny-plugins/userbg/>)\n5, [UserPFP](<https://bunny.nexpid.xyz/usrpfp/>)\n6. [Free Stickers plugin.](<https://aliernfrog.github.io/vd-plugins/FreeStickers/>)\n7. [Nitro Themes plugin.](<https://nullpo1nt3d.github.io/VendettaPlugins/NitroThemes/>)"
```

> *nitro features for free*-\n1

This ends the first sentence with "-", is that intended?

```suggestion
free_nitro = "## List of plugins that provide *nitro features for free*:\n1. [Freemoji plugin](<https://rico040.github.io/bunny-plugins/freemoji/>)\n2. [Realmoji plugin](<https://redstonekasi.github.io/vendetta-plugins/realmoji/>)\n3. [Fake Profile Themes And Effects plugin](<https://ryan-0324.github.io/vendetta-plugins/FakeProfileThemesAndEffects/>)\n4. [UserBG plugin](<https://rico040.github.io/bunny-plugins/userbg/>)\n5, [UserPFP](<https://bunny.nexpid.xyz/usrpfp/>)\n6. [Free Stickers plugin](<https://aliernfrog.github.io/vd-plugins/FreeStickers/>)\n7. [Nitro Themes plugin](<https://nullpo1nt3d.github.io/VendettaPlugins/NitroThemes/>)"
```

---

nope, changed it correctly this time.